packages/signalstory/src/lib/store.ts

```diff
@@ -222,7 +222,22 @@ export class Store<TState> {
   }
 
   /**
-   * Runs a store query potentially targeting many differnt stores with the provided arguments and returns the result.
+   * Creates a computed signal that selects a value from the store's state. This is useful
+   * for creating derived state and binding it into the component tree.
+   *
+   * Use the `runQuery` method if you are targeting multiple stores.
+   *
+   * @param selectFunction A function to select the computed value from the store's state.
+   * @typeparam T The type of the computed signal result.
+   * @typeparam TState The type of the store's state.
+   * @returns The computed signal.
+   */
+  public computed<T>(selectFunction: (state: TState) => T): Signal<T> {
+    return computed(() => selectFunction(this._state()));
```

Comment:
Does this need to be readonly if it's in the computed function? I'm assuming no and I've removed that overhead but perhaps I should add this either way?

Reply:
It should not be forced to be readonly here.
The client can do this manually (using explicit typing or type assertions) or just go for ImmutableStore which will use deep readonly for the state (and derived state) implicitly.